runtime/src/test_utils.rs

```diff
@@ -360,6 +360,11 @@ impl<BS> MockRuntime<BS> {
             actor_exit: Default::default(),
         }
     }
+
+    /// Enable logging to enviornment. Returns error if already init.
+    pub fn init_logging(&self) -> Result<(), log::SetLoggerError> {
```

Comment:
Hm. Actually, why is this a method on the mock runtime?

Reply:
This could be put in Runtime, but I was being careful not to introduce anything new there.

Reply:
hm, ok.

Reply:
actually, at least defining it as part of test_utils simplifies features/dependencies